src/index.js

```diff
@@ -12,6 +12,17 @@ function suggestionTemplate (result) {
   return result && '<strong>' + result.name + '</strong>' + path
 }
 
+// If enabled, this function provides the GOV.UK Registers team with information about your hostname in order to track the component's usage
+function initGoogleAnalyticsTracking () {
+  ga('create', {
+    trackingId: 'UA-90200549-4',
+    cookieDomain: '_gaCookieLocationPickerComponent',
+    name: 'openregisterLocationPicker',
+  });
+
+  ga("set", "anonymizeIp", true);
```

Comment:
Could we be extra careful and add the following? https://github.com/alphagov/govuk_frontend_toolkit/pull/401/files#diff-8b93af583c1a7ca5e9c48a89ebce2b38R18